hrm/department_app/views/employee_views.py

```diff
@@ -0,0 +1,40 @@
+from rest_framework import viewsets
```

Comment:
Please read the documentation regarding ViewSets carefully, you don't need to implement all of these methods implemented below.

Reply:
I already fixed it.